styles/templates/template1/_settings.scss

```
@@ -6,6 +6,7 @@
 @import 'templates/template1/parts/portrait-settings.scss';
 @import 'templates/template1/parts/appendix-settings.scss';
 @import 'templates/template1/parts/index-settings.scss';
+@import 'templates/template1/parts/references-settings.scss';
```

```suggestion
@import 'templates/template1/parts/references-settings';
```
If you don't mind making the change for the other imports too, that would be nice :)